system/registry-cache/templates/deployment.yaml

```diff
@@ -38,6 +39,14 @@ spec:
 {{- end }}
         ports:
         - containerPort: {{ .Values.service.internalPort }}
+        livenessProbe:
+          exec:
+            command:
+              - sh
+              - -c
+              - md5sum /conf/tls.crt | cmp /tmp/tls.crt.md5
+          initialDelaySeconds: 60
+          periodSeconds: 15
```

Comment:
I think this can be relaxed to at least once per minute.

Reply:
I changed this.